tools/cldr-code/src/test/java/org/unicode/cldr/unittest/LikelySubtagsTest.java

```diff
@@ -373,7 +375,7 @@ public void TestMissingInfoForScript() {
         TreeSet<String> sorted = new TreeSet<>(
             ScriptMetadata.getScripts());
         Set<String> exceptions2 = new HashSet<>(
-            Arrays.asList("zh_Hans_CN"));
+            Arrays.asList("zh_Hans_CN", "mww_Hmng_LA", "hnj_Hmnp_US"));  // TODO: script metadata has the reverse?
```

Comment:
@macchiati Script Metadata seems to prefer `hnj+Hmng` and mww+Hmnp but the community wanted to contribute `hnj_Hmnp`

Reply:
Why did this cause a test failure? Even if Hmnp defaults to mww, there should be no problem with having data for other language+script combinations.

Reply:
added an exception